src/components/PathwayCourse/index.js

```diff
@@ -0,0 +1,227 @@
+import React, { useEffect, useState } from "react";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import CheckIcon from "@mui/icons-material/Check";
+import useStyles from "./styles";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import { PATHS, interpolatePath } from "../../constant";
+import { METHODS } from "../../services/api";
+import { useParams } from "react-router-dom";
+import { breakpoints } from "../../theme/constant";
+import { useSelector, useDispatch } from "react-redux";
+import { actions as pathwayActions } from "../PathwayCourse/redux/action";
+import {
+  Container,
+  Box,
+  Grid,
+  Card,
+  CardContent,
+  Typography,
+  CardMedia,
+  Toolbar,
+} from "@mui/material";
+
+const pathways = {
+  1: {
```

Comment:
Currently I get an error when trying to navigate to a pathway page (e.g., for Python with a `pathwayId` of `39` because on line 139, it tries to access `pathways[39].pathway` and pathways[39]` is `undefined` so each of these numbers should be changed to the correct pathway ids.  But is there are reason this information is being hard-coded instead of retrieved from the database?  Is this a temporary fix?
```suggestion
  39: {
```

Reply:
No, actually in v1 Python pathwayId is 39 and in v2 Python pathwayId is 1. Backend doesn't have v2 in production yet. It may merge into production within 2 days or else we have to go with v1 then we won't get the error.